packages/contracts-bedrock/test/kontrol/deployment/KontrolDeployment.sol

```diff
@@ -4,74 +4,12 @@ pragma solidity ^0.8.0;
 import { Deploy } from "scripts/deploy/Deploy.s.sol";
 
 contract KontrolDeployment is Deploy {
-    function runKontrolDeployment() public stateDiff {
-        deploySafe("SystemOwnerSafe");
-        setupSuperchain();
-
-        // deployProxies();
-        deployERC1967Proxy("OptimismPortalProxy");
-        deployERC1967Proxy("L2OutputOracleProxy");
-        deployERC1967Proxy("SystemConfigProxy");
-        deployL1StandardBridgeProxy();
-        deployL1CrossDomainMessengerProxy();
-        deployERC1967Proxy("L1ERC721BridgeProxy");
-        transferAddressManagerOwnership(); // to the ProxyAdmin
-
-        // deployImplementations();
-        deployOptimismPortal();
-        deployL1CrossDomainMessenger();
-        deployL2OutputOracle();
-        deploySystemConfig();
-        deployL1StandardBridge();
-        deployL1ERC721Bridge();
-
-        // initializeImplementations();
-        initializeSystemConfig();
-        initializeL1StandardBridge();
-        initializeL1ERC721Bridge();
-        initializeL1CrossDomainMessenger();
-        initializeOptimismPortal();
+    function runKontrolDeployment() public {
+        runWithStateDiff();
```

Comment:
This is so nice and clean now 😍

Definitely supportive of a future PR that gets rid of this file